index.html

```diff
@@ -535,18 +531,6 @@ <h5 class="modal-title">Please complete the payment process to finalize registra
   document.getElementById("register_button").addEventListener("click", function(event) {
     var form = document.getElementById("main_form");
 
-    // Custom checks
```

Comment:
This part is being removed because not needed anymore, we have removed all attendance fields